sz/Traffic/src/traffic/relogo/UserPatch.groovy

```diff
@@ -11,8 +11,7 @@ import repast.simphony.relogo.schedule.Setup;
 import traffic.ReLogoPatch;
 
 class UserPatch extends ReLogoPatch{
-	static laneWidth = 4
-	
+
```

Comment:
This parameter is purely presentational (does not affect dynamics of the simulation), so I don't think it's beneficial to keep it as a configurable parameter. What do you think?

Reply:
Sure!